tests/by-util/test_comm.rs

```diff
@@ -404,7 +409,7 @@ fn unintuitive_default_behavior_1() {
     scene
         .ucmd()
         .args(&["defaultcheck_unintuitive_1", "defaultcheck_unintuitive_2"])
-        .succeeds()
+        .fails()
```

Comment:
Here it might be useful to have a comment that the behavior differs from GNU `comm`, which doesn't fail.

Reply:
are you sure? it fails for me

```
$ echo "m\nh\nn\no\nc\np\n" > defaultcheck_unintuitive_1
$ echo "m\nh\nn\no\np\n" > defaultcheck_unintuitive_2
$ /usr/bin/comm defaultcheck_unintuitive_1 defaultcheck_unintuitive_2
```

Reply:
I think the reason it fails for you is that `echo` adds a trailing newline by default.

Reply:
oh, right. Comment added :)